R/toolReadSetFromGDX.R

```diff
@@ -0,0 +1,38 @@
+#' toolReadSetFromGDX
+#'
+#' For the given fulldata gdx file, this function returns a dataframe with the
+#' values_parameters of some variables from fulldata.
+#'
+#' @return Dataframe with the values_parameters of some variables
+#' from fulldata.
+#'
+#' @author Anastasis Giannousakis Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- toolReadSetFromGDX()
+#' }
+#'
+#' @importFrom gdx readGDX
+#'
+#' @export
+
+toolReadSetFromGDX <- function() {
```

Comment:
the function should have arguments such as which set to read, and from which GDX. the way it is implemented now it has very limited use

Reply:
set and gdx file name are added